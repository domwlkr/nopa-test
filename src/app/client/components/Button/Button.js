import React, {PropTypes} from 'react';
import { Link } from 'react-router';

const Button = (props) => {
  return (
    <Link {...props} className="button">
      {props.children}
    </Link>
  );
};

Button.propTypes = {
  buttonClass: PropTypes.string
};

export default Button;
