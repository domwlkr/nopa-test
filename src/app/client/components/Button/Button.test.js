import React from 'react';
import Button from './Button';
import renderer from 'react-test-renderer';

test('Renders correctly', () => {
  const tree = renderer.create(
    <Button />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
