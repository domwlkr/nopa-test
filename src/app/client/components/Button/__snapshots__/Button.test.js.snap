// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<a
  className="button"
  onClick={[Function]}
  style={Object {}}
/>
`;
